roles/multibench_run/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: "Ensure tmp directory existes"
+  ansible.builtin.file:
+    state: directory
+    path: "{{ ansible_env.TMPDIR }}"
+    mode: "0755"
+  when: ansible_env.TMPDIR is defined
```

Comment:
will it create the folders in a recursive way? like `mkdir -p`? the problem is that there is no user dciteam in multibench vm, so as a workaround I've done `mkdir -p /home/dciteam/tmp`

Reply:
This is a good observation, if this is having a problem in a non-localhost then it sounds like the ansible_env.TMPDIR should be only available in `localhost`, :thinking: I wonder if that requires a change in the app-agent instead?